vhdbuilder/packer/test/windows-vhd-content-test.ps1

```diff
@@ -289,12 +291,12 @@ function Test-ValidateSinglePackageSignature {
         Remove-Item -Path $installDir -Force -Recurse
     }
 
-    $AllNotSignedFiles = (echo $AllNotSignedFiles | Format-Table | Out-String)
-    Write-Output "All not signed file in cached packages are: $AllNotSignedFiles"
+    $AllNotSignedFiles = (echo $AllNotSignedFiles | ConvertTo-Json -Compress)
```

Comment:
could we only log $AllNotSignedFiles when it is not empty as line 297?

Reply:
Added the condition.